### ISSUE TITLE
Diagram sizing

### ISSUE THREAD

**Issue by wenjing** [trustoverip/TechArch#1]
In section Introduction, Figure 1 is for the ToIP Dual Stack conceptual diagram. The diagram needs to be resized and centered.
In addition, the renderer is not displaying the diagram.

**Comment by talltree** [comment]
@wenjing All makes sense. Should we raise a separate issue that we will most likely also want to update this diagram once this spec is complete? I haven't tracked the suggestions closely, but we've discussed a few tweaks that we've said we'd like to incorporate so the diagram is aligned (at a high level) with the spec.

**Comment by wenjing** [comment]
Yes, that would be a separate issue if you meant the update of that diagram itself since that would be more than just a style change.

**Comment by wenjing** [comment]
This is a demo example of committing a fix with reference to the issue #1 and then close it.
